charts/apps/plausible-analytics/Chart.yaml

```
@@ -1,5 +1,11 @@
 annotations:
-  artifacthub.io/changes: "- kind: changed\n  description: update plausible repository to new location - ghcr.io/plausible/community-edition \n- kind: changed\n  description: update clickhouse to 24.3.3.102-alpine\n"
+  artifacthub.io/changes: |
+    - kind: changed
+      description: add new needed values for plausible 2.1.0
+    - kind: changed
+      description: update plausible to 2.1.0
+    - kind: changed
+      description: update clickhouse chart to 3.9.0
```

FYI: `artifacthub.io/changes` is displayed as related to each specific release, so you need to replace the whole value with the latest changes. I've also added a note about fixing `extraEnv` in the template.

```suggestion
  artifacthub.io/changes: |
    - kind: changed
      description: add new needed values for plausible 2.1.0
    - kind: fixed
      description: fix templating of extraEnv
```